pyinteraph/path_analysis.py

```diff
@@ -0,0 +1,481 @@
+import argparse
+import logging as log
+import numpy as np
+import networkx as nx
+import MDAnalysis as mda
+import itertools
+import re
+
+def build_graph(fname, pdb = None):
+    """Build a graph from the provided matrix"""
+
+    try:
+        data = np.loadtxt(fname)
+    except:
+        errstr = "Could not load file {:s} or wrong file format."
+        raise ValueError(errstr.format(fname))
+    # if the user provided a reference structure
+    if pdb is not None:
+        try:
+            # generate a Universe object from the PDB file
+            u = mda.Universe(pdb)
+        except Exception:
```

Comment:
You could allow the propagation of the message of the underlying exception caught when creating the Universe by using:
`except Exception as e` and then `f"Exception caught during creation of the Universe : {e}."` when formatting the message of the newly raised exception.
This way whoever may be calling `build_graph` would have a clearer picture of what went wrong in creating the Universe :)